lib/matplotlib/figure.py

```diff
@@ -1523,6 +1524,190 @@ def subplots(self, nrows=1, ncols=1, sharex=False, sharey=False,
                 .subplots(sharex=sharex, sharey=sharey, squeeze=squeeze,
                           subplot_kw=subplot_kw))
 
+    @staticmethod
+    def _normalize_grid_string(layout):
+        layout = inspect.cleandoc(layout)
+        return [list(ln) for ln in layout.strip('\n').split('\n')]
+
+    def subplot_grid(self, layout, *, subplot_kw=None, gridspec_kw=None,
+                     empty_sentinel='-'):
+        """
+        Build a layout of Axes based on 2D layout.
+
+        This is a helper function to build complex GridSpec layouts visually.
+
+        .. note ::
+
+           This API is provisional and maybe revised in the future based on
```

Comment:
I believe this is meant to be "may be revised"

Reply:
:sheep:   I swear English is my native language...